database/migrations/1674400358011_roles.ts

```diff
@@ -0,0 +1,21 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+import Roles from 'App/Enums/Roles'
+
+export default class extends BaseSchema {
+  protected tableName = 'roles'
+
+  public async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.string('name').primary()
+    })
+    this.defer(async (db) => {
+      await db
+        .table(this.tableName)
+        .multiInsert([{ name: Roles.USER }, { name: Roles.MODERATOR }, { name: Roles.ADMIN }])
+    })
+  }
```

Comment:
Kannst du nen kommentar dazuschreiben was das macht? Und warum das da ist?

Reply:
die Funktion ist da um daten zu migrieren wenn sich was am model ändert, kann aber auch verwendet werden um default values in die datenbank zu erstellen

https://github.com/adocasts/user-role-authentication-example/blob/main/database/migrations/1667756114567_roles.ts

Reply:
Ich meinte im Code :)
Oder im Wiki, wie du magst :)

Reply:
im code ist es drin